cli/sub_command.go

```diff
@@ -110,6 +111,7 @@ func configureSubCommand(app commandHost) {
 	// To be done - check for streams with WorkQueue, then prompt with or allow with override --force=WorkQueueDelete
 	// act.Flag("ackPolicy", "Acknowledgment policy (none, all, explicit) (requires JetStream)").Default("none").EnumVar(&c.ackPolicy, "none", "all", "explicit")
 	act.Flag("match-replies", "Match replies to requests").UnNegatableBoolVar(&c.match)
+	act.Flag("match-replies-timeout", "Timeout after which messages without a reply are printed").PlaceHolder("TIMEOUT").DurationVar(&c.matchTimeout)
```

Comment:
I'd love to set this to a (low?) default.

Reply:
Rather just use the overall timeout set with --timeout, dont think its needed add more ways to set timeouts.  That one already have a default.

Still, I am not sure if I want to add this, will give it some thought.